prow/repoowners/repoowners.go

```diff
@@ -261,6 +261,13 @@ type RepoOwner interface {
 
 var _ RepoOwner = &RepoOwners{}
 
+type RepoOwnerWithAliases interface {
+	RepoOwner
+	OwnersAliases() RepoAliases
+}
+
+var _ RepoOwnerWithAliases = &RepoOwners{}
+
```

Comment:
@MadhavJivrajani, understanding if I am reading the changes correctly:

Defining `RepoOwnerWithAliases` both (i) as an interface, (ii) as a variable of type `&RepoOwners{}` – is it to maintain backward compatibility for the plugins that are already using `RepoOwner` from the repoowners package?

Reply:
This is to ensure that the `RepoOwners` struct always implements the `RepoOwnerWithAliases` interface. Places that already use `RepoOwner` interface will be unaffected regardless of this line. This line ensures that compilation will fail if for some reason in the future `RepoOwners` stops implementing our inerface - which will break things because we accept the interface everywhere.